client/scenes/lobby_gui.cpp

```diff
@@ -646,16 +646,16 @@ void scenes::lobby::gui_settings()
 		const auto current = config.resolution_scale;
 		const auto width = stream_view.recommendedImageRectWidth;
 		const auto height = stream_view.recommendedImageRectHeight;
-		auto intScale = int(current * 100);
+		auto intScale = int(current * 10);
 		const auto slider = ImGui::SliderInt(
 		        _("Resolution scale").append("##resolution_scale").c_str(),
 		        &intScale,
-		        50,
-		        350,
-		        fmt::format(_F("%d%% - {}x{} per eye"), int(width * current), int(height * current)).c_str());
+		        5,
+		        35,
+		        fmt::format(_F("{}0%% - {}x{} per eye"), intScale, int(width * current), int(height * current)).c_str());
```

Comment:
For this change it would be nice to do the change in localized messages as well.
And since we're doing the formatting ourselves, we may as well pass intScale * 10 and remove the hardcoded 0, so that we can change it to 5% increments if we want.